csv_download.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+$client_curl = curl_init();
+
+curl_setopt_array($client_curl, array(
+    CURLOPT_URL => "https://api.cookiebot.com/auth/v1/connect/token",
+    CURLOPT_SSL_VERIFYPEER => false,
+    CURLOPT_RETURNTRANSFER => true,
+    CURLOPT_CUSTOMREQUEST => "POST",
+    CURLOPT_POSTFIELDS => "grant_type=client_credentials&client_id=" . get_option('client_ID') . "&client_secret=" . get_option('client_Secret'),
+    CURLOPT_HTTPHEADER => array(
+        "Content-Type: application/x-www-form-urlencoded"
+    ),
+));
+
+$client_response = curl_exec($client_curl);
+
+curl_close($client_curl);
+
+$token = json_decode($client_response);
+
+
+
+
+
+$CSV_curl = curl_init();
+
+curl_setopt_array($CSV_curl, array(
+CURLOPT_URL => "https://app-cookiebot-api-umbracodashboard-prod.azurewebsites.net/urls/" . get_option('domainID') . "?format=json",
+CURLOPT_RETURNTRANSFER => true,
+CURLOPT_SSL_VERIFYPEER => false,
+CURLOPT_ENCODING => "",
+CURLOPT_MAXREDIRS => 10,
+CURLOPT_TIMEOUT => 0,
+CURLOPT_FOLLOWLOCATION => true,
+CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+CURLOPT_CUSTOMREQUEST => "GET",
+CURLOPT_HTTPHEADER => array(
+    "Authorization: Bearer " . $token->access_token
+),
+));
+
+$CSV_response = curl_exec($CSV_curl);
+
+curl_close($CSV_curl);
+
+$CSV = json_decode($CSV_response);
+
+$fileName = 'somefile.csv';
+
+header("Cache-Control: must-revalidate, post-check=0, pre-check=0");
+header('Content-Description: File Transfer');
+header("Content-type: text/csv");
+header("Content-Disposition: attachment; filename={$fileName}");
+header("Expires: 0");
+header("Pragma: public");
+
+$fh = @fopen( 'php://output', 'w' );
+
+$headerDisplayed = false;
+ 
+foreach ( $CSV->crawlUrls as $data ) {
+    $data_array = array(
```

Comment:
💡 Try to use variable-names that describe meaning or intention, rather than implementation details.

That is, every variable in a program is "data", every array is an "array" - using these words as names does not help other programmers understand the meaning of the values or the intention of your code to do something with those values.

For example, a better name for `$data` might be `$url`, so we don't forget what type of data is in there - `$data_array` might be `$columns` or `$csv_values`, hinting at the intent to emit these values to CSV, and so on.

(Also, try to keep with the code standards of the project you're contributing to: don't mix `camelCaps` and `snake_case` - it usually increases the chance of an open-source project maintainer accepting your pull-request, as they tend to have many contributors, and therefore are usually trying to maintain a consistent codebase.)